inngest-core/src/main/kotlin/com/inngest/Inngest.kt

```diff
@@ -1,18 +1,39 @@
 package com.inngest
 
-class Inngest(val appId: String) {
-    // TODO - Fetch INNGEST_EVENT_KEY env variable on instantiation
+import com.beust.klaxon.Klaxon
+import java.io.IOException
 
-//    fun send(event: Event): EventAPIResponse {
-//        val requestBody = Klaxon().toJsonString(event)
-//        val client = HttpClient.newBuilder().build()
-//        val request =
-//            HttpRequest.newBuilder()
-//                .uri(URI.create("http://localhost:8288/e/"))
-//                .POST(HttpRequest.BodyPublishers.ofString(requestBody))
-//                .build()
-//        val response = client.send(request, HttpResponse.BodyHandlers.ofString())
-//        val body = Klaxon().parse<EventAPIResponse>(response)
-//        return body;
-//    }
-}
+class Inngest
+    @JvmOverloads
+    constructor(val appId: String, framework: String? = null) {
```

Comment:
Framework needs to be inferred from the adapter and is never a value passed in on the constructor.

https://www.inngest.com/docs/reference/client/create